bitnami/postgresql-ha/templates/postgresql/statefulset.yaml

```diff
@@ -321,6 +321,22 @@ spec:
             - name: REPMGR_DATABASE
               value: {{ (include "postgresql-ha.repmgrDatabase" .) | quote }}
             {{- end }}
+            {{- if .Values.postgresql.repmgrFenceOldPrimary }}
+            - name: REPMGR_FENCE_OLD_PRIMARY
+              value: {{ .Values.postgresql.repmgrFenceOldPrimary | quote }}
+            {{- end }}
+            {{- if .Values.postgresql.repmgrChildNodesCheckInterval }}
+            - name: REPMGR_CHILD_NODES_CHECK_INTERVAL
+              value: {{ .Values.postgresql.repmgrChildNodesCheckInterval | quote }}
+            {{- end }}
+            {{- if .Values.postgresql.repmgrChildNodesConnectedMinCount }}
+            - name: REPMGR_CHILD_NODES_CONNECTED_MIN_COUNT
+              value: {{ .Values.postgresql.repmgrChildNodesConnectedMinCount | quote }}
+            {{- end }}
+            {{- if .Values.postgresql.repmgrChildNodesDisconnectTimeout }}
+            - name: REPMGR_CHILD_NODES_DISCONNECT_TIMEOUT
+              value: {{ .Values.postgresql.repmgrChildNodesDisconnectTimeout | quote }}
+            {{- end }}
```

Comment:
I am missing all this new parameters in the `values.yaml`.

Reply:
hi.. not sure if I follow... these are optional parameters, you can set it to certain value depending on your needs, I have documented their usage in `README.md` for postgresql-ha. 
When you need them however you can set it in the `values.yaml` file. 

Here's a brief description of their usage:

`REPMGR_FENCE_OLD_PRIMARY` : set this to `true` if you want to have the cluster killing the old/false primary upon its return into the network, realizing that its children are no longer following it. Otherwise, you can leave it unset. Essentially a switch for the feature introduced in this PR.

`REPMGR_CHILD_NODES_CHECK_INTERVAL` : set this to certain interval in integer seconds, say 5, and that will be the interval that `repmgr` on the primary node will use to poll for the child nodes currently following the primary node. Default by `repmgr` is 5 seconds.

`REPMGR_CHILD_NODES_CONNECTED_MIN_COUNT` : this is the minimum number of child nodes following the primary node required to consider the said primary node as current primary. Should the number of child nodes found to fall below this value, the primary node will be considered as false primary (since by binary reasoning, a child node not following this primary node must have found a new primary elsewhere, hinting at a new primary elected in the absence of this primary node) . Default is 1, meaning by default, `repmgr` will only consider the node a false primary after all of its child node had been found disconnected (found a new primary).

`REPMGR_CHILD_NODES_DISCONNECT_TIMEOUT` : time to wait in seconds once the number of child nodes connected fell below the minimum count required, before `repmgr` call the command set in `child_nodes_disconnect_command` in `repmgr.conf`, which is to kill the false primary in this PR. Default is 30 seconds.   


A more detail documentation of parameters used can be found [here](https://repmgr.org/docs/current/repmgrd-primary-child-disconnection.html).

Hopefully that helps. :)

Reply:
What he means is that all values that are used in the chart template should be documented in the values.yaml file. The table in the README.md is auto-generated from the information in the values.yaml file